.pre-commit-config.yaml

```diff
@@ -15,18 +15,20 @@ repos:
           - "eslint-plugin-cypress"
           - "eslint-plugin-jest"
           - "eslint-plugin-react-hooks"
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v4.0.0-alpha.8
-    hooks:
-      - id: prettier
-        files: '\.[jt]sx?$' # *.js, *.jsx, *.ts, *.tsx
+  # more linting and formatting
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.5.0
     hooks:
     - id: check-merge-conflict
-    - id: check-yaml
     - id: trailing-whitespace
+    - id: check-yaml
     - id: end-of-file-fixer
+    - id: trailing-whitespace
```

Comment:
Am i reading this right? Theres two hooks for trailing-whitespace now?

Reply:
Huh. I thought I had fixed that in a previous PR. Maybe it got messed up during merge conflict resolution? 🤷